ffbs-parker-nodeconfig/files/usr/sbin/nodeconfig.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/busybox sh
+# !!!!!!
+# THIS SCRIPT MUST HAVE GID 800 (gluon-mesh-vpn)
+# USE `gluon-wan nodeconfig.sh`
+# !!!!!!
+SIGN_PUB_KEY=/etc/parker/node-config-pub.key
+DEFAULT_SLEEP=20
+export LIBPACKETMARK_MARK=1
+
+tmpdir=/tmp/ff-Ohb0ba0u/
+mkdir -p "$tmpdir"
+
+version="$(cat /lib/gluon/release)"
+
+LOGGER="logger -s -t nodeconfig.sh"
+$LOGGER "Starting up. I am running ${version}"
+
+if [ ! -s /etc/parker/wg-privkey ]; then
+    $LOGGER No Wireguard keys found, generating...
+    mkdir -p /etc/parker
+    rm -f /etc/parker/wg-pubkey
+    wg genkey > /etc/parker/wg-privkey.tmp
+    wg pubkey < /etc/parker/wg-privkey.tmp > /etc/parker/wg-pubkey
+    sync
+    mv /etc/parker/wg-privkey.tmp /etc/parker/wg-privkey
+    $LOGGER Wireguard keys generated
+else
+    $LOGGER Wireguard keys found, proceeding
+fi
+
+pubkey=$(sed 's/+/%2B/g' /etc/parker/wg-pubkey)
+
+while true; do
+    vpn_enabled=$(uci get gluon.mesh_vpn.enabled)
+    if [ "$vpn_enabled" == 0 ]; then
+        sleep $DEFAULT_SLEEP
+        continue
+    fi
+    rm -f "${tmpdir}config.json" "${tmpdir}config.json.sig"
+    nonce=""
+    while [ -z "${nonce}" ]; do
+        nonce=$(head -c16 /dev/urandom | md5sum | head -c32)
+    done
+    v6mtu="$(cat /proc/sys/net/ipv6/conf/br-wan/mtu)"
+    if ! ip route show table 1 | grep -Fq via ; then
+        $LOGGER no WAN route. Doing nothing.
+        sleep $DEFAULT_SLEEP
+        continue;
+    fi
+
+    config_server=$(uci get parker.nodeconfig.config_server)
```

Comment:
It would be nice if we can add a check here if this config exists, to transparently send a log message in case someone broke their UCI config.

Something like
```suggestion
    if ! config_server=$(uci get parker.nodeconfig.config_server); then
        $LOGGER unable to get config_server from uci
        continue
    fi
```

Reply:
@SmithChart still open

Reply:
See https://github.com/SmithChart/community-packages/pull/7

Reply:
@grische can you verify that this has been resolved by https://github.com/SmithChart/community-packages/pull/7 ?